src/Text/LLVM/PP.hs

```diff
@@ -338,9 +353,15 @@ ppGlobalAttrs hasValue ga
         --   * initialized structure
         --   * external scalar
         --   * external structure
+        ppAddrSpace (gaAddrSpace ga) <+>
         constant
```

Comment:
The comments above this ought to be updated, since it claims that only the value (and nothing else) is printed. Really, it ought to say that the address space and the value are printed now.